### ISSUE TITLE
Check exit codes of external programs

### ISSUE THREAD

**Issue by bernt-matthias** [KennthShang/PhaBOX#35]
It would be great if the code could check the exit code of external program calls, e.g. [here](https://github.com/KennthShang/PhaBOX/blob/e01dc7a39e25838576073c1a6d4fc77c00a0533d/src/phabox2/phatyp.py#L136) and phabox return a non-zero exit code in case one of the calls fails.

**Comment by KennthShang** [comment]
Thanks for the advice. I will update this in the future version.

Best,
Jiayu

**Comment by KennthShang** [comment]
I have updated the codes with the latest release version 2.1.4.

In addition, I set up a new pull request for PhaBOX 2.1.4 to Bioconda [#51958](https://github.com/bioconda/bioconda-recipes/pull/51958).

Because I always got a problem during the test: `[Errno 28] No space left on device `. I decide to remove all the stand-alone packages from the receipt and let the users install them independently. Hope it will work this time.

Best,
Jiayu
